app/assets/stylesheets/provider/_buttons.scss

```diff
@@ -164,3 +164,19 @@ form.formtastic .button-bar, .button-bar {
   padding: line-height-times(1/4) line-height-times(1/3) !important;
   height: line-height-times(5/4);
 }
+
+/****** HACK ******/
+// This fixes missing PF4
```

Comment:
Add a TODO? so this can be grepped later?

Reply:
This is an alternative to TODO we've been using for a while since @didierofrivia introduced it. If you grep `* HACK` you'll see there are some already.

Reply:
It started because we will modifying vendor code, mainly patternfly and webpacker

Reply:
OK but please share back to the community by:

1. Opening an issues in the upstream
2. Opening a PR
3. Point the link in this PR so there are cross-references

Reply:
The way I see it, there are already tags all over the codebase: TODO, FIXME, HACK... They exists as a reminder that something needs to be done, eventually. There is no point in creating JIRAs or issues that will be closed a month after because nobody took them.

IMHO what we should do is to regularly (once a month maybe) take a look at the existing tags and address 1 or 2. This is a globally accepted convention so it should be easy for the community to identify and fix them.

Reply:
I mean open an issue in PF4 not here :)